src/formatDate.js

```diff
@@ -8,7 +8,69 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat[3];
```

Comment:
The separator is assumed to be at index 3 of the `fromFormat` array, which may not be correct for all date formats. Consider dynamically determining the separator based on the actual format provided.